RUFAS/weather.py

```diff
@@ -200,42 +138,17 @@ def get_conditions_series(self, time: Time, starting_offset: int, ending_offset:
             Series of current day conditions in chronological order.
 
         """
-        current_date = Utility.convert_ordinal_date_to_month_date(time.current_calendar_year, time.current_julian_day)
-        date_series = Utility.generate_time_series(current_date, starting_offset, ending_offset)
-
-        starting_year_index = time.current_simulation_year - (current_date.year - date_series[0].year)
-        starting_day_index = date_series[0].toordinal() - date(date_series[0].year, 1, 1).toordinal() + 1
-        ending_year_index = time.current_simulation_year + (date_series[-1].year - current_date.year)
-        ending_day_index = date_series[-1].toordinal() - date(date_series[-1].year, 1, 1).toordinal() + 1
-        conditions_series = []
-        for year in range(starting_year_index, ending_year_index + 1):
-            if starting_year_index == ending_year_index:
-                start_day = starting_day_index
-                end_day = ending_day_index
-            elif year == starting_year_index:
-                start_day = starting_day_index
-                end_day = len(self.__mean_daily_temperature[year]) - 1
-            elif year == ending_year_index:
-                start_day = 0
-                end_day = ending_day_index
-            else:
-                start_day = 0
-                end_day = len(self.__mean_daily_temperature[year]) - 1
+        condition_list = []
```

Comment:
And other refs to this var.
```suggestion
        conditions_list = []
```